src/Game.java

```diff
@@ -0,0 +1,64 @@
+import java.util.*;
+
+public class Game {
+    private List<String> words = new ArrayList<>();
+    Random random = new Random();
+    WordsDatabase database = new WordsDatabase(words, random);
```

Comment:
Creating objects like this should be done in the constructor. Nonetheless I dont like the idea of passing words and random do WordsDatabase why can't we have those properties in that class?

Reply:
You're right, while changed code i miss it.